static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -495,11 +492,9 @@ const makeEpicABTest = ({
 };
 
 const buildEpicCopy = (row: any, hasCountryName: boolean) => {
```

Comment:
did we consider searching the copy for `%%COUNTRY_NAME%%` rather than having this dependency between the boolean and the presence of the template string?

Reply:
yes, it's partly to avoid scanning through lots of text for each test, and partly because we want to exclude users from tests where `hasCountryName == true` if they're not in one of the supported countries